storage/client.go

```diff
@@ -68,6 +74,20 @@ type AzureStorageServiceError struct {
 	RequestID                 string
 }
 
+type odataErrorMessageMessage struct {
+	Lang  string `json:"lang"`
+	Value string `json:"value"`
+}
+
+type odataErrorMessageInternal struct {
```

Comment:
@MindFlavor  do you ever use the error message/code in generating a human-readable error? perhaps we should return a typed-error that people can cast into and read these fields?